.vs/ProjectSettings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "CurrentProjectSetting": null
```

Comment:
I will do that if I have to do another commit, thanks. For now I just pulled them again.